aries_vcx/src/common/ledger/transactions.rs

```diff
@@ -124,6 +133,54 @@ pub async fn into_did_doc(profile: &Arc<dyn Profile>, invitation: &Invitation) -
     Ok(did_doc)
 }
 
+fn _ed25519_public_key_to_did_key(public_key_base58: &str) -> VcxResult<String> {
```

Comment:
Optional: move `did:key` stuff to separate file

Reply:
at this point I don't know very well where to create this class, could you indicate where to establish this.

Reply:
It's alright :-) We'll move it later. Didn't give it much thought where exactly it shall go